### PR TITLE
Feature/connector responses

### PR DESCRIPTION
**Pull request by dhirama**
#### What is the purpose of this pull request?
Add **connector response** information to `PaymentMethod` component for **Multibanco** payments.

#### What problem is this solving?
The missing **connector response** information is mandatory for the customer to proceed with **Multibanco** payments.

#### How should this be manually tested?
Order with **Multibanco** payment:
https://dandraft--btpt.myvtex.com/checkout/orderPlaced/?og=942671578240

Order without **Multibanco** payment:
https://dandraft--btpt.myvtex.com/checkout/orderPlaced/?og=946272145689

#### Screenshots or example usage
image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- **connector response** information to `PaymentMethod` component for **Multibanco** payments.
+- Generic `context.json` file to `messages` folder.
+- **en**, **es** and **pt** translations for **Multibanco** labels.
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- **connector response** information to `PaymentMethod` component for **Multibanco** payments.
+- Generic `context.json` file to `messages` folder.
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
```

File: messages/context.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (salesfelipe) [inline messages/context.json]: No need for context.

File: react/components/Payment/ConnectorResponseInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          defaultMessage: label,
+        })}
+        :
+      </span>
+      <span className="more-info__data__item__value">{value}</span>
+    </div>
+  )
+}
+
+ConnectorResponseInfo.propTypes = {
```
Comment (salesfelipe) [inline react/components/Payment/ConnectorResponseInfo.tsx]: No need for the propTypes

File: react/components/Payment/ConnectorResponseInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+const ConnectorResponseInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  label,
+  value,
+  intl,
+}) => {
+  return (
+    <div className="more-info__data__item f7">
+      <span className="more-info__data__item__label mr2 fw5">
+        {intl.formatMessage({
```
Comment (salesfelipe) [inline react/components/Payment/ConnectorResponseInfo.tsx]: Use the FormattedMessage component instead and doing so you can remove the inject intl.

File: react/components/Payment/ConnectorResponseInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import React, { FunctionComponent } from 'react'
+import { injectIntl, InjectedIntlProps, intlShape } from 'react-intl'
+
+const ConnectorResponseInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  label,
+  value,
+  intl,
+}) => {
+  return (
+    <div className="more-info__data__item f7">
```
Comment (salesfelipe) [inline react/components/Payment/ConnectorResponseInfo.tsx]: This css classes should be added trhough cssModules

Comment (victorhmp) [reply react/components/Payment/ConnectorResponseInfo.tsx]: Yep, I don't think these CSS classes are defined right now

File: react/components/Payment/ConnectorResponseInfo.tsx (hunk 4 above)
Comment (victorhmp) [reply react/components/Payment/ConnectorResponseInfo.tsx]: @dhirama with TypeScript, there is no need for PropTypes, as the type checks are made in compile time

**Pull request merged**